srcalc/src/main/java/gov/va/med/srcalc/web/view/ReferenceItem.java

```diff
@@ -75,6 +76,11 @@ public String getHelpTextAsHtml()
      */
     public List<String> getReferenceInfo()
```

Comment:
Why is this a List at all? Since this is all view code, we can assume tight coupling between the DisplayItem classes and enterVariables.jsp and just make it a single String.